src/db/z_db.erl

```diff
@@ -213,9 +221,11 @@ with_connection(F, Context) ->
 	end.
 
 
+-spec assoc_row(string(), #context{}) -> list(tuple()).
```

Comment:
The spec in line 224 uses a record, please define a type for the record and use that instead.